docs/features/sessions/4_pause.md

```diff
@@ -33,12 +33,21 @@ The equivalent via the CLI
 grid session resume the-interactive-name
```

Comment:
Standard should be to use "SESSION NAME" instead of "the-interactive-name" to avoid confusion